features/footer.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Footer
+  As a user
+  In order to see information about the site
+  I would like to see info about the platform company in the footer
+
+
+  Scenario: Display footer to visitor
+    Given I visit the landing page
+    Then "footer" is present
```

Comment:
is it? or `.. should be present` or `visible`?  Revisit this please.

Reply:
Changed to `Then "footer" is visible`